src/components/TabSelector/TabSelector.js

```diff
@@ -38,23 +38,35 @@ const defaultProps = {
 
 const getIcon = (route) => {
     switch (route) {
+        case CONST.TAB.MANUAL:
+            return Expensicons.Pencil;
         case CONST.TAB.SCAN:
             return Expensicons.Receipt;
+        case CONST.TAB.NEW_CHAT:
+            return Expensicons.User;
+        case CONST.TAB.NEW_ROOM:
+            return Expensicons.Hashtag;
         case CONST.TAB.DISTANCE:
             return Expensicons.Car;
         default:
-            return Expensicons.Pencil;
+            throw new Error(`Route ${route} has no icon set.`);
     }
 };
 
 const getTitle = (route, translate) => {
     switch (route) {
+        case CONST.TAB.MANUAL:
+            return translate('tabSelector.manual');
         case CONST.TAB.SCAN:
             return translate('tabSelector.scan');
+        case CONST.TAB.NEW_CHAT:
+            return translate('tabSelector.chat');
+        case CONST.TAB.NEW_ROOM:
+            return translate('tabSelector.room');
         case CONST.TAB.DISTANCE:
             return translate('common.distance');
         default:
-            return translate('tabSelector.manual');
+            throw new Error(`Route ${route} has no title set.`);
     }
 };
```

Comment:
Just an idea, instead of two long `switch`es, we could create one function called e.g. `getIconAndTitle` returning an object like:
```javascript
{
   icon: Expensicons.Pencil;
   title: translate('tabSelector.manual');
}

// ...

const {icon, title} = getIconAndTitle(route, translate);
```

Reply:
Makes sense now that it grew significantly